config/routes.rb

```diff
@@ -17,6 +17,9 @@
   namespace :admin do
     get '/dashboard', to: "dashboard#index"
     get '/users', to: "dashboard#show_users"
+    get '/merchants', to: "dashboard#show_merchants"
+    get 'merchant/:id/disable', to: "dashboard#disable_merchant"
+    get 'merchant/:id/enable', to: "dashboard#enable_merchant"
```

Comment:
Going with Kiera's suggestions we could even do the following:

```suggestion
    get '/dashboard', to: "dashboard#index"
    get '/users', to: "dashboard#show"
    get '/merchants', to: "dashboard#show"
    get 'merchant/:id/disable', to: "dashboard#destroy"
    get 'merchant/:id/enable', to: "dashboard#create"
```

This reminds me of the section in the README that tells us about how "Not everything can be fully destroyed". Not sure if the above suggestion is more REstful or not.

Reply:
Oops I just realized my suggestion doesn't work for the show view.